drivers/wifi/nrf700x/osal/fw_if/umac_if/src/cmd.c

```diff
@@ -169,6 +169,7 @@ enum wifi_nrf_status umac_cmd_init(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 #endif /* !CONFIG_NRF700X_RADIO_TEST */
 
 	umac_cmd_data->op_band = op_band;
+	umac_cmd_data->maxidle_insec = CONFIG_NRF700X_BSS_MAX_IDLE_TIME;
```

Comment:
Shouldn't this be coming from WPA supplicant?

Reply:
Looks like supplicant doesn't support setting bss_max_idle_time for station. Also, current requirement is to pass the max idle timeout at bootup. So passing it here seems correct.

Reply:
It does support but it is not passed as part of `wpa_driver_associate_params` in `associate` callback and passing it in the bootup is not quite right, it has to be part of the associate/connect command.